GVRf/Framework/framework/src/main/java/org/gearvrf/GVRJassimpAdapter.java

```diff
@@ -626,17 +635,21 @@ private void importLights(List<AiLight> lights, Hashtable<String, GVRLightBase>
     }
 
     private void setLightProp(GVRLightBase gvrLight, AiLight assimpLight){
-        gvrLight.setFloat("attenuation_constant", assimpLight.getAttenuationConstant());
-        gvrLight.setFloat("attenuation_linear", assimpLight.getAttenuationLinear());
-        gvrLight.setFloat("attenuation_quadratic", assimpLight.getAttenuationQuadratic());
+        //gvrLight.setFloat("attenuation_constant", assimpLight.getAttenuationConstant());
```

Comment:
I screwed up and commented out the attenutation settings. Put them back in and you are good to go.